src/views/Stats.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <VContainer>
+    <VRow>
+      <VCol>
+        <h1 class="text-h4">{{ $t("common.stats") }}</h1>
+      </VCol>
+      <VCol class="d-flex justify-end">
```

Comment:
Maybe I'm missing something, but shouldn't this be a `VRow` to make this horizontal?

Reply:
A `VRow` can't be a direct child of itself (or at least, that doesn't look ok).
To use a VRow, it would have to be
```html
<VCol>
  <VRow>
    <VCol>
      <VSwitch />
     </VCol>
     <VCol>
       <DateRangeSelect />
     </VCol>
```
But that felt like a lot of unnecessary structure for to apply two css rules